Add overset/underset whatsnew entry (#19497)
matplotlib/matplotlib: doc/users/next_whats_new/mathtext.rst

``matplotlib.mathtext`` now supports *overset* and *underset* LaTeX symbols
---------------------------------------------------------------------------

`.mathtext` now supports *overset* and *underset*, called as 

> ``\overset{body}{annotation}`` or ``\underset{body}{annotation}``, where 

Reviewer: OK, but now this is wrong again, right?

Author: Yeah, I blindly copied the suggestion, let me fix that

``matplotlib.mathtext`` now supports *overset* and *underset* LaTeX symbols
---------------------------------------------------------------------------

> `.mathtext`, the default TeX layout engine which is shipped along with
> Matplotlib now supports symbols like *overset* and *underset*.
> 
> The structure which should be followed: "\overset{body}{annotation}" or
> "\underset{body}{annotation}", where *body* would be the text "above" or
> "below" the *annotation* - the baseline character.

Reviewer: ```suggestion
`.mathtext` now supports *overset* and *underset*, called as 
``\overset{body}{annotation}`` or ``\underset{body}{annotation}``, where 
*annotation* is the text "above" or "below" the *body*.
```
